labelbox/data/serialization/coco/panoptic_dataset.py

```diff
@@ -70,20 +77,32 @@ def process_label(label: Label,
         for annotation_idx, annotation in enumerate(annotations[class_name]):
             categories[annotation.name] = hash_category_name(annotation.name)
             if isinstance(annotation.value, Mask):
-                segment, canvas = (mask_to_coco_segment_info(
+                coco_segment_info = mask_to_coco_segment_info(
```

Comment:
Questions around this.
1. How can coco_segment_info be set to `segment, canvas` when it's only returning `SegmentInfo` object?
2. When can coco_segment_info be None?

Reply:
1. mask_to_coco_segment_info returns both of those things. It is hard to read because canvas looks like it is part of the SegmentInfo object but it isn't 
2. coco_segment_info can be None if the annotation is invalid. Like an empty seg mask is exported for whatever reason.